### PR TITLE
Updated seed file

### PR DESCRIPTION
**Pull request by somedayrainbows**
* Rewrote the seed file to be a seed class; left previously commented-out code alone

@subelsky

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -85,14 +85,14 @@ group :development, :test do
   gem 'spring-commands-rspec'
   gem 'rb-fsevent' #osx file system changes
   gem 'factory_girl_rails', require: false
-  gem 'faker'
+  gem 'database_cleaner'
+  gem 'faker', :git => 'git://github.com/stympy/faker.git', :branch => 'master'
```
Comment (subelsky) [inline Gemfile]: @somedayrainbows can you add a comment here about why we need to use the forked version of this gem? Also please switch to Ruby 2+ style keyword hash (`git: "git:/...", branch: 'master'`)

File: db/seeds.rb [hunk 2]
```diff
@@ -1,30 +1,54 @@
...
+# require_relative "../spec/factories/websites"
+
+class Seed
+
+  def initialize
+    generate_groups
+    generate_metum
+    generate_status
+    generate_admin_user
+  end
```
Comment (subelsky) [inline db/seeds.rb]: People using this class might not expect the initializer to have side-effects. Can you change this method to be  a separate `.generate` method? Also recommend adding a short comment above the class definition stating what the purpose of the class is.

**Comment by sinabahram** [comment]
Thanks for this work. wherever possible, can we please delete commented out code? It is such a bad practice, IMHO, and it’s why we have things like GIT in the first place.

**Comment by subelsky** [comment]
@somedayrainbows  and I weren't sure if you wanted to keep that stuff around for future reference, no problem

**Comment by sinabahram** [comment]
Thanks for that, but no need to keep. If it were up to me, I’d search for any instance of commented out code and immediately delete it. if one wishes to see previous code, one can revisit old commits, and properly rebased pull requests. This is sort of a pet peeve right now since I’m dealing with similar cleanup on an unrelated project, where old code was just scattered hap hazard all over.

**Comment by somedayrainbows** [comment]
Closing this as it's now outdated. @subelsky

**Comment by subelsky** [comment]
I prefer to say it was "overtaken by events" 😀

**Comment by somedayrainbows** [comment]
😁

**Pull request closed without merging**